base/ca/shared/profiles/ca/caServerCertWithCRLDP.cfg

```diff
@@ -0,0 +1,98 @@
+desc=This certificate profile is for enrolling server certificates and CRL-DP extention for certificate validation.
```

Comment:
same comments from the ECC copy applies to this profile.
Also, we need to include CMC copies for the same changes.